### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by andrewdotn**

### CODE REVIEW

File: docs/dictionary-data.md [hunk 1]
```diff
@@ -201,9 +122,13 @@ The fields are:
...
-    that people would like to specify things such as source-language text
-    to be shown in the current orthography, cross-references, notes,
+    If we were starting from scratch we might call the `definition` field a
+    `displayDefinition`, but we already have some data, and often it is the
+    only one definition field provided.
+
+    The definition must currently be a single unformatted string. We are
+    aware that people would like to specify things such as source-language
+    text to be shown in the current orthography, cross-references, notes,
     examples, and so on.
```
Comment (dwhieb) [inline docs/dictionary-data.md:132]: This suggestion more reflects what I'd _hope_ would be the case. I think it'd get really bad if we kept trying to cram more things into the definition fields.

```suggestion
    The definition must currently be a single unformatted string. We are
    aware that people would like to specify things such as source-language
    text to be shown in the current orthography, cross-references, notes,
    examples, and so on. If these are supported in the future, they will
    likely be separate fields rather included within the definition.
```

File: docs/linguistic-data-intro.md [hunk 2]
```diff
@@ -5,6 +5,31 @@ the morphodict code to use.
...
+    codes](https://iso639-3.sil.org/code_tables/639/data) for the source
+    and target languages.
+
+  - The definition of a single word.
+
+  - A paradigm table for that word. This could be as simple as a photocopy
+    of a page from a textbook showing each wordform, annotated with the
+    desired FST analysis for each.
+
+  - It’s nice to know the endonym, i.e., the name of the language in the
```
Comment (dwhieb) [inline docs/linguistic-data-intro.md:21]: I've noticed some people know _autonym_ but not _endonym_. They mean essentially the same thing.

```suggestion
  - It’s nice to know the endonym, i.e., the autonym or name of the language in the
```

File: docs/linguistic-data-intro.md [hunk 3]
```diff
@@ -133,3 +158,89 @@ by making sure every row has the same number of tabs.
...
+Having the following information available in some form is *extremely
+useful* for developers, community members, and other linguists who are not
+as familiar with the language, or your conventions for describing it, as
+you are.
+
+This may be in the form of prose documents, spreadsheets, diagrams, and so
+on, as long as the intent is to provide reference material for humans to
+understand what is going on.
+
+  - **Part-of-speech categories** and the relations among them
```
Comment (dwhieb) [inline docs/linguistic-data-intro.md:173]: Would here be a good place for that screenshot of the part-of-speech ontology we took?

File: docs/linguistic-data-intro.md [hunk 4]
```diff
@@ -133,3 +158,89 @@ by making sure every row has the same number of tabs.
 
 By default it will reformat every `altlab.tsv` file, but you can explicitly
 give it the path to any TSV file on the command line.
+
+# Abstract data
+
+Having the following information available in some form is *extremely
+useful* for developers, community members, and other linguists who are not
+as familiar with the language, or your conventions for describing it, as
+you are.
```
Comment (dwhieb) [inline docs/linguistic-data-intro.md:167]: 💯

File: docs/linguistic-data-intro.md [hunk 5]
```diff
@@ -133,3 +158,89 @@ by making sure every row has the same number of tabs.
...
+    categories with varying levels of detail, e.g., “nouns” and “verbs” and
+    “type-1 animate intransitive verbs.” They also group these categories
+    into other meta-categories, such as ‘part of speech’ or ‘word class’ or
+    ‘linguistic category.’ For example, they may say that ‘nouns’ and
+    ‘verbs’ are ‘parts of speech’ while ‘transitive verbs’ and
+    ‘intransitive verbs’ are ‘word classes’ within the ‘verb’ ‘part of
+    speech.’
+
+    As handy as it would be, there is no universal terminology here, so
+    please spell out what terminology you’re using!
```
Comment (dwhieb) [inline docs/linguistic-data-intro.md:185]: So glad you included this.

File: docs/linguistic-data-intro.md [hunk 6]
```diff
@@ -133,3 +158,89 @@ by making sure every row has the same number of tabs.
...
+      - A prose description of the category.
+
+      - What are some example members of this category?
+
+      - Are the members declinable?
+
+        If the members of this category are in theory declinable, but for
+        practical purposes morphodict will not be doing any declining—maybe
+        nobody’s figured out the rules yet, or the rules are known but
+        nobody’s put them into an FST—it is useful to note that.
```
Comment (dwhieb) [inline docs/linguistic-data-intro.md:222]: declension = nouns
conjugation = verbs
inflection = either

If we just use "decline", readers might not realize they need to do this for verbs or other categories (e.g. pronouns, articles, etc.) too.

```suggestion
      - Can the members be declined or conjugated? (i.e., Do they have inflections?)

        If the members of this category are in theory inflectable, but for
        practical purposes morphodict will not be doing any inflecting—maybe
        nobody’s figured out the rules yet, or the rules are known but
        nobody’s put them into an FST—it is useful to note that.
```

File: docs/linguistic-data-intro.md [hunk 7]
```diff
@@ -133,3 +158,89 @@ by making sure every row has the same number of tabs.
...
+
+      - Do all the members of this category share the same paradigm table
+        template, with the same FST tags but different FST lemma inputs?
+
+        If the answer to this question is ‘no,’ morphodict cannot use this
+        particular part-of-speech category as a paradigm, and you can skip
+        the other questions for this category.
+
+      - If the members of the category are declinable, then, for some
```
Comment (dwhieb) [inline docs/linguistic-data-intro.md:231]:
```suggestion
      - If the members of the category are inflectable, then, for some
```

File: docs/linguistic-data-intro.md [hunk 8]
```diff
@@ -133,3 +158,89 @@ by making sure every row has the same number of tabs.
...
+        example members, please provide one or more paradigm tables as
+        might be found in instructional or reference material.
+
+        These are extremely useful for checking that morphodict is
+        displaying paradigms correctly correctly.
+
+      - If the members are declinable, what is the preferred
```
Comment (dwhieb) [inline docs/linguistic-data-intro.md:238]:
```suggestion
      - If the members are inflectable, what is the preferred
```

File: docs/linguistic-data-intro.md [hunk 9]
```diff
@@ -133,3 +158,89 @@ by making sure every row has the same number of tabs.
...
+        practical purposes morphodict will not be doing any declining—maybe
+        nobody’s figured out the rules yet, or the rules are known but
+        nobody’s put them into an FST—it is useful to note that.
+
+      - Do all the members of this category share the same paradigm table
+        template, with the same FST tags but different FST lemma inputs?
+
+        If the answer to this question is ‘no,’ morphodict cannot use this
+        particular part-of-speech category as a paradigm, and you can skip
+        the other questions for this category.
```
Comment (dwhieb) [inline docs/linguistic-data-intro.md:229]: This is basically what Arok did for Plains Cree—he divided the categories into subcategories based on the details of how they inflect.

```suggestion
        If the answer to this question is ‘no,’ morphodict cannot use this
        particular part-of-speech category as a paradigm, and you can skip
        the other questions for this category. You could however create
        subcategories, each with their own inflectional paradigm.
```

**Review by dwhieb — APPROVED** [review]

File: docs/linguistic-data-intro.md (hunk 2 above)
Comment (andrewdotn) [reply docs/linguistic-data-intro.md:21]: Thanks!

**Pull request merged**